cmake/modules/OmrDetectSystemInformation.cmake

```diff
@@ -83,11 +83,13 @@ macro(omr_detect_system_information)
 
    if(OMR_HOST_OS STREQUAL "zos")
       # ZOS specifics
+      set(OMR_TOOLCONFIG "gnu")
```

Comment:
yeah I wrote in the commit, this setting is wrong, but at least there's something here while we work on proper xlc support.